docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md

```diff
@@ -0,0 +1,52 @@
+5. Open the `storage1` created at step 2
+6. Click ***Show attributes*** button 
+7. Click ***Request file system access*** link 
+8. Repeat steps 5-7 for `storage2`
+9. Logout
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select any tool | |
+| 4 | Launch tool with *Default settings* | |
+| 5 | At the **Runs** page, click the just-launched run | |
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the `pipe storage mount -f -m 775 -o allow_other <destination_folder>` command | |
+| 9 | Enter and perform the `echo "test info" >> <destination_folder>/storage1/file3` command | |
+| 10 | Logout and _Check audit log_ (see below) | Record <br> `WRITE cp://storage1/file3` <br> is shown in the top of logs records |
+| 11 | Repeat steps 5-9 for `cp <destination_folder>/storage1/file1 <destination_folder>/storage2/file1` command | Records <br>`READ cp://storage1/file1` <br> `WRITE cp://storage2/file1` <br> are shown in the top of logs records |
+| 12 |Repeat steps 5-9 for `cat <destination_folder>/storage1/file2` command | Records <br>`READ cp://storage1/file1` <br> is shown in the top of logs records |
+| 13 | Repeat steps 5-9 for `mv <destination_folder>/storage1/file2 <destination_folder>/storage2` command | Record <br> `MOVE cp://storage1/file2 cp://storage2/file2` <br> is shown in the top of logs records |
+| 14 | Repeat steps 5-9 for `rm <destination_folder>/storage2/file2` command | Record <br> `DELETE cp://storage2/file2` <br> is shown in the top of logs records |
+| 15 | Repeat steps 5-9 for `cp -rf <destination_folder>/storage1/folder1 <destination_folder>/storage2/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ cp://storage1/folder1/<file>` <br> `WRITE cp://storage2/folder1/<file>` <br> are shown in the top of logs records |
+| 16 | Repeat steps 5-9 for `mv -f <destination_folder>/storage1/folder2 <destination_folder>/storage2/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ cp://storage1/folder2/<file>` <br> `WRITE cp://storage2/folder2/<file>` <br> `DELETE cp://storage2/folder2/<file>` <br> are shown in the top of logs records |
+| 17 | Repeat steps 5-9 for `rm -f  <destination_folder>/storage1/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) record <br> `DELETE cp://storage1/folder2/<file>` <br> are shown in the top of logs records |
+
+
+\* _Check audit log_:
+1. Login as the admin user from the prerequisites | |
+2. Open the **Settings** page | |
+3. Open the **SYSTEM MANAGEMENT** page | |
+4. Select the **LOGS** tab | |
+5. Click ***Show advanced*** button | |
+6. Click **Type** field and select `audit` value from drop-down
```

Comment:
I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.